modules/apps/login/login-web/src/main/resources/META-INF/resources/login.jsp

```diff
@@ -180,11 +180,13 @@
 			<%@ include file="/navigation.jspf" %>
 		</div>
 
-		<aui:script sandbox="<%= true %>">
```

Comment:
This script should be inline so that it is triggered as soon as possible. Otherwise it is deferred to the end of the page which can cause weird interactions.

I'm not 100% sure this is needed for the fix, I added it when trying another solution, but I think it is safer in any case.